tests/tf/test_core.py

```diff
@@ -141,6 +138,42 @@ def test_simple_model(ecommerce_data: SyntheticData):
     )
 
 
+def test_block_to_model(ecommerce_data: SyntheticData):
+    model = ml.MLPBlock([64]).to_model(
```

Comment:
Similar comment: Better to showcase how a user can define embedding_dim here.

Reply:
Added it [here](https://github.com/NVIDIA-Merlin/models/blob/tf/wrap-as-model/tests/tf/test_core.py#L142-L147), works the same for `Model.from_block`